releasenotes/notes/pulse-round-a014390e414c79c8.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes a bug in RZXCalibrationBuilder that was causing pulses to sometimes be
```

Comment:
```suggestion
    Fixes a bug in the :class:`~.RZXCalibrationBuilder` transpiler pass that was causing pulses to sometimes be
```

Reply:
Thanks. I'm curious, what does the `~.` do?

Reply:
The `~` limits the reference text to just the class name `RZXCalibrationBuilder` instead of `passes.RZXCalibrationBuilder`. The `.` prefix just changes how sphinx searches it does a suffix match instead of an exact match search, it might not be strictly necessary in this case I just find it a bit easier to work with in general. You can see the documentation for these in the text at the end of this section: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=%3Aclass%3A#cross-referencing-python-objects, right before "The C Domain" header (we also don't need the `:py:` prefix from the docs because it's all python by default).